packages/containers/src/Notification/Notification.sagas.js

```diff
@@ -22,6 +22,9 @@ export function* onPushNotification(action) {
 }
 
 function* onCMFError(action) {
+	if (process.env.RELEASE_WITH_ERROR && process.env.NODE_ENV === 'production') {
```

Comment:
By default, it is not disabled. I prefer disable by default.
In other hand, we can activate when  process.env.NODE_ENV !== 'production' to remove the errors during the development and see them

Reply:
What we have validated with PM is: Dev must see error notification during dev.
Because it s a temporary solution we dont want team to activate it but to deactivate it.
Either we will have same scenario at the end when we will want to remove the flag.

OK to rename the flag